platform/windows/os_windows.cpp

```diff
@@ -1599,7 +1599,7 @@ String OS_Windows::get_system_dir(SystemDir p_dir, bool p_shared_storage) const
 
 	PWSTR szPath;
 	HRESULT res = SHGetKnownFolderPath(id, 0, nullptr, &szPath);
-	ERR_FAIL_COND_V(res != S_OK, String());
+	ERR_FAIL_COND_V_MSG(res != S_OK, String(), "missing system directory " + OS::system_dir_to_string(p_dir));
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(res != S_OK, String(), vformat("Missing system directory: %s.", OS::system_dir_to_string(p_dir)));
```

Reply:
New to receiving a review for a Godot PR. What's the etiquette? Add the fix locally, squash commit, force push?

Reply:
I'd say to fix locally, ammend with `git commit --amend` and then force push